rio_discord_bot/discord_client.py

```diff
@@ -0,0 +1,35 @@
+import discord
+
+
+class DiscordClient(discord.Client):
+    def __init__(self, guild: int, channel: int, players: list):
+        self.guild: int = guild
```

Comment:
```suggestion
    def __init__(self, channel: int, players: list):
```

the guild is not used so worth removing it.